File: C#/openai login design/UserControlSignUp.cs
using System.Windows.Forms;

namespace openai_login_design
{
    public partial class UserControlSignUp
    {
        public UserControlSignUp()
        {
            InitializeComponent();
        }
        private void LinkLabel1_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Clear();
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Add(new UserControlLogin());
        }
    }
}

File: VB/openai login design/UserControlSignUp.vb
Public Class UserControlSignUp
    Private Sub LinkLabel1_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles LinkLabel1.LinkClicked
        Form1.Guna2Panel1.Controls.Clear()
        Form1.Guna2Panel1.Controls.Add(New UserControlLogin())
    End Sub
End Class
